index.d.ts

```diff
@@ -0,0 +1,36 @@
+declare module 'sw-toolbox' {
+  interface Cache {
+    name: string
+    maxEntries: number
+    maxAgeSeconds: number
+  }
+  interface Options {
+    debug: boolean
+    networkTimeoutSeconds: number
+    cache: Cache
+  }
+  interface Handler {
+    (): void
```

Comment:
A valid handler must return a `Promise` for a `Request`.

It also receives a `Request` as an argument.